index.html

```diff
@@ -334,77 +363,52 @@ <h3>
           following algorithm:
         </p>
         <ol>
-          <li>Let <var>window</var> be the {{Window}} object of the
-          <a>top-level browsing context</a> for which the user agent will
-          attempt installation.
+          <li>Let <var>manifest</var> and <var>manifest URL</var> be the values
+          that were created during <a>steps to determine installability of the
+          document</a>.
           </li>
-          <li>Then, <a>in parallel</a>:
+          <li>If <var>manifest URL</var> exists, and the result of running <a>
+            processing the `serviceworker` member</a> with <var>manifest</var>
+            returns a valid <var>registration</var>, the user agent MAY:
             <ol>
-              <li>Instantiate an <a>installation process</a>.
-              </li>
-              <li>Let <a>manifest</a> and <a>manifest URL</a> be the result of
-              <a>obtaining the manifest</a>.
-              </li>
-              <li>If <a>obtaining the manifest</a> results in an error, a user
-              agent can, at this point, fall back to using the <a>top-level
-              browsing context</a> {{Document}}'s metadata to populate an
-              <a>installation process</a>'s UI.
-              </li>
-              <li>If <a>obtaining the manifest</a> succeeds, and the result of
-              running <a>processing the `serviceworker` member</a> with
-              <a>manifest</a> returns a valid <var>registration</var>, a user
-              agent can at this point:
-                <ol>
-                  <li>Let <var>client</var> be the <a>top-level browsing
-                  context</a> {{Document}}'s <a>relevant settings object</a>,
-                  or <code>null</code> if unavailable.
-                  </li>
-                  <li>Invoke <a>Start Register</a> with <var>scope</var> and
-                  <var>src</var> members of the <var>registration</var>, a new
-                  <var>promise</var>, <var>client</var>, <a>manifest URL</a>,
-                  plus the <var>type</var> and <var>update_via_cache</var>
-                  members of the <var>registration</var>, in which case the
-                  state of the settled <var>promise</var> determines whether
-                  the <a>installation succeeded</a> or not.
-                  </li>
-                </ol>
-              </li>
-              <li>If the <a>installation succeeded</a>, <a>queue a task</a> on
-              the <a>application life-cycle task source</a> to <a>fire an
-              event</a> named <code>appinstalled</code> at the
-              <var>window</var> object.
+              <li>Let <var>client</var> be the <a>top-level browsing
+              context</a> {{Document}}'s <a>relevant settings object</a>, or
+              <code>null</code> if unavailable.
+              </li><!-- Start Register no longer exists in the Service Worker
```

Comment:
Do we have some other alternative if Start Register is gone?

Reply:
Yeah but I'll look at this as part of #789.